resources/com.f3d-app.F3D.metainfo.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-    <id>com.kitware.F3D</id>
+    <id>com.f3d-app.F3D</id>
```

Comment:
@Meakk : this should already be in master ? not sure what to make of that.

Reply:
Yes, I'm not sure why it appears here.

Reply:
Try to merge it and we'll see.

Reply:
alright

Reply:
commit is visible two times in the historic, but file is perfectly fine. Well, lets not forget to merge into release when needed :)